### ISSUE TITLE
feat: integrate NoSql DB (Realm) for local data storage

### ISSUE THREAD

**Issue by sankur-codes** [sankur-codes/PassMan#1]
#### Points To Remember
* Create flexible DB structure to make it easy to enhance later
* Main recycleview should have only title. Rest data should be in childView

**Comment by Rajnish-Lmeter** [comment]
Hi, I want to work on all the issues. All are linked to one another. Should I first start with Database Integration (Realm)? Because all other issues needed CRUD operation. 
Please let me know. @sankur-codes

**Comment by sankur-codes** [comment]
> 
> 
> Hi, I want to work on all the issues. All are linked to one another. Should I first start with Database Integration (Realm)? Because all other issues needed CRUD operation.
> Please let me know. @sankur-codes

Yes all issues are linked to one another as the application is in initial stages of development phase.
You can start with this issue.

Assigning the issue to @Rajnish-Lmeter

**Comment by Rajnish23** [comment]
Sorry, I used the wrong Github account. Please reassign the issue to this account.
